saleor/product/views.py

```diff
@@ -120,18 +121,20 @@ def category_index(request, path, category_id):
                         category_id=category_id)
     products = (products_with_details(user=request.user)
                 .filter(categories__name=category)
-                .order_by(DEFAULT_SORT))
+                .order_by('name'))
     product_filter = ProductFilter(
         request.GET, queryset=products, category=category)
     products_paginated = get_paginator_items(
         product_filter.qs, PAGINATE_BY, request.GET.get('page'))
     products_and_availability = list(products_with_availability(
         products_paginated, request.discounts, request.currency))
-    sort_by = request.GET.get('sort_by', DEFAULT_SORT)
+    sort_by = request.GET.get(
+        'sort_by', pgettext_lazy('Product field', 'name'))
     ctx = {'category': category, 'filter': product_filter,
            'products': products_and_availability,
            'products_paginated': products_paginated,
-           'sort_by_choices': SORT_BY_FIELDS,
-           'sort_by_label': sort_by.strip('-'),
+           'sort_by_choices': get_sort_by_choices(product_filter),
+           'now_sorted_by': pgettext_lazy(
+               'Product field', sort_by.strip('-')),
```

Comment:
Why do you pass `Product field` as context here? This seems wrong, the field itself uses `Sort by label` as context which also seems wrong. Both should probably use something like `Product list sorting form`.